docs/02_cleos/03_command-reference/create/account.md

```diff
@@ -29,16 +50,14 @@ Options:
   --delay-sec UINT            set the delay_sec seconds, defaults to 0s
 ```
 
-## Command
-A set of EOSIO keys is required to create an account. The EOSIO keys can be generated by using `cleos create key`.
+## Requirements
+For prerequisites to successfully run this command, see the **Before you Begin** section of the [_How to Create a Blockchain Account_](../02_how-to-guides/how-to-create-an-account.md) guide.
 
-```sh
-cleos create account inita tester EOS4toFS3YXEQCkuuw1aqDLrtHim86Gz9u3hBdcBw5KNPZcursVHq EOS7d9A3uLe6As66jzN8j44TXJUqJSK3bFjjEEqR4oTvNAB3iM9SA
-```
+## Example Command
 
-## Output
+```console
+cleos create account inita tester EOS4toFS3YXEQCkuuw1aqDLrtHim86Gz9u3hBdcBw5KNPZcursVHq EOS7d9A3uLe6As66jzN8j44TXJUqJSK3bFjjEEqR4oTvNAB3iM9SA
```

Comment:
does this need the where bit to explain the parameters - help explain the 2 public keys